pkg/plugins/resources/dockerfile/condition_test.go

```diff
@@ -58,10 +60,7 @@ func TestDockerfile_Condition(t *testing.T) {
 					"matcher": "golang",
 				},
 			},
-			mockTest: text.MockTextRetriever{
-				Exists: false,
-			},
-			wantErr: fmt.Errorf("the file NOTEXISTING.Dockerfile does not exist"),
+			wantErr: fmt.Errorf("The file NOTEXISTING.Dockerfile does not exist."),
```

Comment:
```suggestion
			wantErr: fmt.Errorf("the file NOTEXISTING.Dockerfile does not exist"),
```

Reply:
Linter complain about error message starting with capital letter and ending with punctuation point. I guess is because error message can be merged